### ISSUE TITLE
Consider to register .phar as executable binary

### ISSUE THREAD

**Issue by sun** [composer/windows-setup#34]
While investigating Windows installation instructions for PHPUnit in https://github.com/sebastianbergmann/phpunit-documentation/issues/218 …

I stumbled over the slightly dated, but very interesting [Python installation instructions for Windows](https://docs.python.org/2/using/windows.html#executing-scripts) that are offering an alternative way that I didn't think of yet:

``` cmd
:: Register the .phar extension
> assoc .phar=PHARFile
:: Set the Open command line
> ftype PHARFile=php.exe "%1" %*
:: Make `script` work without `.phar` extension
> set PATHEXT=%PATHEXT%;.PHAR
```

UAC will block all of those commands when not in an administrative shell, but an actual installer runs in an elevated environment anyway already.

In a nutshell, the commands are doing the following:
1. Create `HKCR\.phar` → `HKCR\PHARFile`
2. Create `HKCR\PHARFile\shell\open\command` = `'php.exe "%1" %*'` [REG_EXPAND_SZ]
3. Extend `HKCU\Environment\PATHEXT` = `'%PATHEXT%;.PHAR'` [REG_EXPAND_SZ]

In turn, `*.phar` is treated like any other binary/script, and the following commands work natively, as soon as a `*.phar` file is located anywhere in `%PATH%`:

``` cmd
> composer
> phpunit
```

Mayhaps worth to investigate?
